torch/csrc/cuda/Stream.h

```diff
@@ -11,8 +12,7 @@ struct THCPStream {
   // the hook for placement-new'ing/delete'ing it; simpler to just
   // rely on the packed representation...)
   uint64_t cdata;
-  int device;
```

Comment:
Removing these two member vars changes the behavior of accessing `device` and `cuda_stream`. Although these two members are not advertised in our doc, will this be a problem for our users? Or should I keep them in `PyMemberDef` and just point them to new locations?

Reply:
I think I should keep those two vars.